boards/arm/atsamd20_xpro/doc/atsamd20_xpro.rst

```diff
@@ -134,7 +134,7 @@ References
 .. target-notes::
 
 .. _Microchip website:
-    www.microchip.com/DevelopmentTools/ProductDetails.aspx?PartNO=ATSAMD20-XPRO
+    https://www.microchip.com/DevelopmentTools/ProductDetails.aspx?PartNO=ATSAMD20-XPRO
```

Comment:
You may need to rebase. You've picked up a stray commit.

Reply:
Done